src/components/SearchResults/Ngo/Details.js

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Descriptions, Table } from "antd";
+import { getString } from "@utils/Parser/strUtils";
+const { Item } = Descriptions;
+
+/**
+ * The columns that has to be shown for the NGO report
+ */
+const columns = [
+  {
+    title: "Zone",
+    dataIndex: "zone",
+    key: "zone",
+    render: (text) => <a>{text}</a>,
+  },
+  {
+    title: "Ward",
+    dataIndex: "ward",
+    key: "ward",
+    render: (ward) => <a>{ward}</a>,
+  },
+  {
+    title: "Pins",
+    dataIndex: "pincodes",
+    key: "pincodes",
+    render: (pincodes) => <a>{pincodes.join(", ")}</a>,
+  },
+];
+
+/**
+ * This function is used to render the expandable result section in the Ngo form
+ * @param {*} param0 The records that are generated from the search query.
+ */
+function Details({ record }) {
+  return (
+    <Descriptions title="Details" bordered size="middle">
+      <Item label="EMail" span={3}>
+        {getString(record.email)}
+      </Item>
+      <Item label="Alternate Mobile Number" span={3}>
+        {getString(record.alt_mob)}
+      </Item>
+      <Item label="Address" span={3}>
+        {record.address}
+      </Item>
+      <Item label="Regions" span={3}>
+        {getRegions(record)}
+      </Item>
+      <Item label="Registration" span={3}>
```

Comment:
The label strings can match the form labels (refer NGOForm) or be a short one, "Registration Number" or "Reg No."

Reply:
Will do the necessary changes.

Reply:
Done